Cargo.toml

```diff
@@ -48,7 +48,7 @@ default-features = false
 features = ["zstd", "snappy"]
 
 [build-dependencies]
-configure_me_codegen = { version = "0.4.4", default-features = false }
+configure_me_codegen = { version = "0.4.6", default-features = false }
```

Comment:
WTF, this is a BS change and not needed.

Reply:
We can [turn off this BS](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#versioning-strategy) but annoyingly that will cause other issues. I don't know which is worse. https://github.com/dependabot/dependabot-core/issues/4009